src/static/riot/competitions/form/_phases.tag

```diff
@@ -291,7 +312,7 @@
 
         self.close_modal = function () {
             $(self.refs.modal).modal('hide')
-            self.clear_form()
+            setTimeout(self.clear_form, 200)
```

Comment:
this timeout seems high, is this necessary?

Reply:
Has to wait for the entirety of the modal to fade. It could probably go a little lower, but even 100 was too fast on my machine, you could still see it switch.

Reply:
Could you use the `onClosed` or-some-such callback here?

https://stackoverflow.com/questions/28870073/semantic-ui-modal-close-ok-cancel-callback

Reply:
Maybe. I'll look into it. Definitely a better idea than `setTimeout`

Reply:
`onHidden` was the way to go 👍🏻